src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/RejectedFilesReportService.java

```diff
@@ -26,23 +23,21 @@ public RejectedFilesReportService(BlobManager blobManager) {
 
     public List<RejectedFile> getRejectedFiles() {
         return blobManager
-            .listRejectedContainers()
+            .getRejectedContainers()
             .stream()
             .flatMap(container ->
                 getBlobs(container)
                     .map(listItem -> new RejectedFile(
-                        FilenameUtils.getName(listItem.getUri().toString()),
-                        container.getName()
+                        listItem.getName(),
+                        container.getBlobContainerName()
                     )))
             .collect(toList());
     }
 
-    private Stream<ListBlobItem> getBlobs(CloudBlobContainer container) {
-        return StreamSupport.stream(
-            container
-                .listBlobs(null, true, EnumSet.of(SNAPSHOTS), null, null)
-                .spliterator(),
-            false
-        );
+    private Stream<BlobItem> getBlobs(BlobContainerClient container) {
+        ListBlobsOptions listOptions = new ListBlobsOptions();
+        listOptions.getDetails().setRetrieveSnapshots(false);
```

Comment:
does existing code retrieves snapshots ?

Reply:
previous code returns snapshots , we should return snapshots